code/modules/surgery/organs/external/tails.dm

```diff
@@ -169,6 +169,19 @@
 	feature_key = "tail_cat"
 	color_source = ORGAN_COLOR_HAIR
 
+/obj/item/organ/external/tail/cat/fox
+	name = "fox tail"
+	bodypart_overlay = /datum/bodypart_overlay/mutant/tail/cat/fox
+	dna_block = null
+	sprite_accessory_override = /datum/sprite_accessory/tails/human/fox
+
+///Fox tail bodypart overlay
+/datum/bodypart_overlay/mutant/tail/cat/fox
+
+/datum/bodypart_overlay/mutant/tail/cat/fox/New()
+	. = ..()
+	set_appearance(/datum/sprite_accessory/tails/human/fox)
```

Comment:
Why this? This should be handled by organ insert

Reply:
I forgot to delete it